docs/devices-api/api-reference/user/get-users-and-providers.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Get Users and Providers"
+description: "Returns a list of users with their IDs and connected providers."
+api: "GET /user/users"
+---
+
+## Response
+
+<ResponseField name="connectedUsers" type="array" required>
+  The user's connected providers
```

Comment:
this isn't accurate, it's an object containing multiple properties - let's describe the response properly

see here for an example https://github.com/metriport/metriport/blob/4036cfc2f4c19c53142ad29658a25eafe9bea59b/docs/medical-api/api-reference/patient/list-patients.mdx